src/gordon_gcp/plugins/enricher.py

```diff
@@ -89,6 +90,7 @@ def _validate_config(self):
             self.config['retries'] = 5
 
     def _init_auth(self):
+        # req keys: scopes, keyfile
```

Comment:
I'm not sure this makes sense to include here, since the validation should already have happened at this point and we know `keyfile` is in the config. `scopes` is not required https://github.com/spotify/gordon-gcp/blob/develop/src/gordon_gcp/clients/auth.py#L80